src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,74 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+interface PaginationProps {
+  pageItemsToShow: number[];
+  pageItemCount: number;
+  currentPage: number;
+  visibleItems: string[];
+  handleChangePage: (choicePage: number) => void;
+}
```

Comment:
The `Pagination` component should be stateless and use the props `total`, `perPage`, `currentPage`, and `onPageChange` as specified in the task description. The current implementation uses different props, which does not meet the task requirements.